apps/login/turbo.json

```diff
@@ -0,0 +1,29 @@
+{
+  "extends": ["//"],
+  "pipeline": {
+    "build": {
+      "outputs": ["dist/**", ".next/**", "!.next/cache/**"],
+      "dependsOn": ["^build"]
+    },
+    "test": {
+      "dependsOn": ["@zitadel/server#build", "@zitadel/react#build"]
```

Comment:
is there a way that these dependencies get automatically resolved from the package? Should not just the `build` resolve to it?

Edit: Can't we use a central .json and associate an order to the pipeline like:
```
"pipeline": {
    "a#build": {
      "dependsOn": ["c#build"]
    },
    "b#build": {},
    "c#build": {
      "dependsOn": ["b#build"]
    }
  }
```

Reply:
Here is the commit that centralizes all turbo config, but I REALLY hate it https://github.com/zitadel/typescript/pull/28/commits/8094daf6660af494385dba1d40975e02becf0dd6.

Let's please define package specific config in the package that should be configured.